### ISSUE TITLE
rnv platform eject not working

### ISSUE THREAD

**Issue by franco948** [pavjacko/renative#445]
I'm trying to execute `rnv platform eject` in order to modify the `build.gradle` file on Android.

When the command is executed I can't select any option and there is a `Disabled` label alongside each option.

Nothing happens when `<a>` or any other key is pressed. There is not interactivity at all.

The only thing I can do is press `Enter` but I just get an error.

_Command line after **rnv platform eject**_
![image](https://user-images.githubusercontent.com/20349612/77577985-843e4c80-6eb6-11ea-95c4-d8b77076295e.png)

_Error after pressing **Enter**_
```
$ rnv platform eject - ERRROR! Cannot read property 'undefined' of undefined
TypeError: Cannot read property 'undefined' of undefined
    at rnvPlatformEject$ (C:\Users\MyUser\AppData\Roaming\npm\node_modules\rnv\src\platformTools\index.js:99:13)
    at tryCatch (C:\Users\MyUser\AppData\Roaming\npm\node_modules\rnv\node_modules\@babel\runtime\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (C:\Users\MyUser\AppData\Roaming\npm\node_modules\rnv\node_modules\@babel\runtime\node_modules\regenerator-runtime\runtime.js:288:22)
    at Generator.prototype.<computed> [as next] (C:\Users\MyUser\AppData\Roaming\npm\node_modules\rnv\node_modules\@babel\runtime\node_modules\regenerator-runtime\runtime.js:114:21)
    at tryCatch (C:\Users\MyUser\AppData\Roaming\npm\node_modules\rnv\node_modules\@babel\runtime\node_modules\regenerator-runtime\runtime.js:62:40)
    at invoke (C:\Users\MyUser\AppData\Roaming\npm\node_modules\rnv\node_modules\@babel\runtime\node_modules\regenerator-runtime\runtime.js:152:20)
    at C:\Users\MyUser\AppData\Roaming\npm\node_modules\rnv\node_modules\@babel\runtime\node_modules\regenerator-runtime\runtime.js:162:13
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

**Comment by xbw12138** [comment]
I had a same problem

**Comment by pavjacko** [comment]
@franco948 @xbw12138 thx for reporting this!
could you try `0.29.1-alpha.8` 

if problem persists (with interactive shell) you can also eject manually by editing your `./renative.json` file as for example following:

```
"paths": {
        "platformTemplatesDirs": {
            "android": "./platformTemplates"
        }
    },
```

but as you're only trying to update single file there is more elegant way to do so by utilising single file override feature.

simply add your custom `build.gradle` file to this location:

`./appConfigs/base/builds/android/build.gradle`
or
`./appConfigs/base/builds/android/app/build.gradle`

depending on which gradle file you're trying to override
it will automatically override default one.

basically anything you place into `./appConfigs/base/builds/android/...` will end up in your final project

same for any other platform
